src/components/Proposal/ProposalSummary/ProposalSummarySchemeRegistrar.tsx

```diff
@@ -33,8 +44,19 @@ export default class ProposalSummary extends React.Component<IProps, IState> {
     this.setState({ network: (await getNetworkName()).toLowerCase() });
   }
 
-  private copySchemeAddressOnClick = (schemeRegistrar: ISchemeRegistrar) => (): void => copyToClipboard(schemeRegistrar.schemeToRegister);
-  private copySchemeParamsHashOnClick = (schemeRegistrar: ISchemeRegistrar) => (): void => copyToClipboard(schemeRegistrar.schemeToRegisterParamsHash);
+  private copySchemeAddressOnClick = (schemeRegistrar: ISchemeRegistrar) => (): void =>
+  {
+    const { showNotification } = this.props;
+    copyToClipboard(schemeRegistrar.schemeToRegister);
+    showNotification(NotificationStatus.Success, "Copied to clipboard!");
+  }
+
+  private copySchemeParamsHashOnClick = (schemeRegistrar: ISchemeRegistrar) => (): void =>
+  {
+    const { showNotification } = this.props;
+    copyToClipboard(schemeRegistrar.schemeToRegisterParamsHash);
+    showNotification(NotificationStatus.Success, "Copied to clipboard!");
```

Comment:
we do not have a generic function to copy to clipboard ?

Reply:
no, I was lamenting that myself

Reply:
what about https://github.com/daostack/alchemy/blob/c13b354f84a8f86e22afc903527508ac8ff9b066/src/components/Notification/Notification.tsx#L38 ?

Reply:
that is just the notification, not the copy-to-clipboard itself.